server/app/auth/oidc/admin/AdfsGroupAccessor.java

```diff
@@ -0,0 +1,37 @@
+package auth.oidc.admin;
+
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+import org.pac4j.oidc.profile.OidcProfile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class AdfsGroupAccessor {
+  private static final Logger logger = LoggerFactory.getLogger(AdfsGroupAccessor.class);
+
```

Comment:
**suggestion:** Could you add a javadoc for this method and the class?

Reply:
Done!